package/python-kiwi-spec-template

```diff
@@ -295,6 +295,10 @@ Provides:       kiwi-filesystem:xfs
 Requires:       dosfstools
 Requires:       e2fsprogs
 Requires:       xfsprogs
+%if ! (0%{?suse_version} && 0%{?suse_version} < 1600)
+Provides:       kiwi-filesystem:erofs
```

Comment:
This still needs to be with the other Provides lines in the "not debbuild" conditional, since these provides are invalid for debian.

Reply:
reading specs is no fun, no indentation between conditions, so hard to read. Thanks